pkg/strategy/elliottwave/draw.go

```diff
@@ -13,47 +13,49 @@ import (
 
 func (s *Strategy) InitDrawCommands(store *bbgo.SerialMarketDataStore, profit, cumProfit types.Series) {
 	bbgo.RegisterCommand("/draw", "Draw Indicators", func(reply interact.Reply) {
-		canvas := s.DrawIndicators(store)
-		if canvas == nil {
-			reply.Message("cannot render indicators")
-			return
-		}
-		var buffer bytes.Buffer
-		if err := canvas.Render(chart.PNG, &buffer); err != nil {
-			log.WithError(err).Errorf("cannot render indicators in ewo")
-			reply.Message(fmt.Sprintf("[error] cannot render indicators in ewo: %v", err))
-			return
-		}
-		bbgo.SendPhoto(&buffer)
+		go func() {
+			canvas := s.DrawIndicators(store)
+			if canvas == nil {
+				reply.Message("cannot render indicators")
+				return
+			}
+			var buffer bytes.Buffer
+			if err := canvas.Render(chart.PNG, &buffer); err != nil {
+				log.WithError(err).Errorf("cannot render indicators in ewo")
+				reply.Message(fmt.Sprintf("[error] cannot render indicators in ewo: %v", err))
+				return
+			}
+			bbgo.SendPhoto(&buffer)
+		}()
 	})
 	bbgo.RegisterCommand("/pnl", "Draw PNL(%) per trade", func(reply interact.Reply) {
-		canvas := s.DrawPNL(profit)
-		var buffer bytes.Buffer
-		if err := canvas.Render(chart.PNG, &buffer); err != nil {
-			log.WithError(err).Errorf("cannot render pnl in drift")
-			reply.Message(fmt.Sprintf("[error] cannot render pnl in ewo: %v", err))
-			return
-		}
-		bbgo.SendPhoto(&buffer)
+		go func() {
+			canvas := s.DrawPNL(profit)
+			var buffer bytes.Buffer
+			if err := canvas.Render(chart.PNG, &buffer); err != nil {
+				log.WithError(err).Errorf("cannot render pnl in ewo")
+				reply.Message(fmt.Sprintf("[error] cannot render pnl in ewo: %v", err))
+				return
+			}
+			bbgo.SendPhoto(&buffer)
+		}()
 	})
 	bbgo.RegisterCommand("/cumpnl", "Draw Cummulative PNL(Quote)", func(reply interact.Reply) {
-		canvas := s.DrawCumPNL(cumProfit)
-		var buffer bytes.Buffer
-		if err := canvas.Render(chart.PNG, &buffer); err != nil {
-			log.WithError(err).Errorf("cannot render cumpnl in drift")
-			reply.Message(fmt.Sprintf("[error] canot render cumpnl in drift: %v", err))
-			return
-		}
-		bbgo.SendPhoto(&buffer)
+		go func() {
+			canvas := s.DrawCumPNL(cumProfit)
+			var buffer bytes.Buffer
+			if err := canvas.Render(chart.PNG, &buffer); err != nil {
+				log.WithError(err).Errorf("cannot render cumpnl in ewo")
+				reply.Message(fmt.Sprintf("[error] canot render cumpnl in ewo: %v", err))
```

Comment:
error log will be redirect to telegram, so you don't need to send message manually.